### PR TITLE
OCPNODE-454: kubelet: ignore reboots on kubelet CA change

### PR DESCRIPTION
**Pull request by rphillips**
**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by romfreiman** [comment]
If you specify jira ticket in the commit message it will be linked automatically

File: pkg/daemon/update.go [hunk 1]
```diff
@@ -467,6 +467,7 @@ func (dn *Daemon) applyOSChanges(oldConfig, newConfig *mcfgv1.MachineConfig) (re
 func calculatePostConfigChangeActionFromFileDiffs(oldIgnConfig, newIgnConfig ign3types.Config) (actions []string) {
 	filesPostConfigChangeActionNone := []string{
 		"/var/lib/kubelet/config.json",
+		"/etc/kubernetes/kubelet-ca.crt",
```
Comment (yuqi-zhang) [inline pkg/daemon/update.go:470]: This by itself probably won't do anything. I've never observed this to change without `/etc/kubernetes/ca.crt` also changing.

Comment (rphillips) [reply pkg/daemon/update.go:470]: `/etc/kubernetes/ca.crt` gets rendered out from the bootstrap to the masters... I do not see any processes reading ca.crt on a fully installed master. The kubelet does not use ca.crt.

I added /etc/kubernetes/ca.crt into the list.

**Comment by rphillips** [comment]
removed the ca.crt ignore.

David's PR fixes that situation: https://github.com/openshift/machine-config-operator/pull/2294

File: pkg/daemon/update.go (hunk 1 above)
Comment (rphillips) [reply pkg/daemon/update.go:470]: With David's patch we can just keep kubelet-ca.crt...

**Comment by rphillips** [comment]
rebased, ready for 4.8

/hold cancel

**Comment by yuqi-zhang** [comment]
I am :+1: on the change generally, just curious, is there a good way for us to test the kubelet change being applied live properly? Is this covered somewhere in the e2e testing (the general live reload functionality)?

**Review by kikisdeliveryservice — CHANGES_REQUESTED** [review]
can you update the commit to contain a little more detail?

**Comment by yuqi-zhang** [comment]
I've included this in https://github.com/openshift/machine-config-operator/pull/2398 and updated the commit message (https://github.com/openshift/machine-config-operator/pull/2398/commits/3aabfec2ece0d7bd3487399e615e27b8efbfe808) while keeping you as the original author. If you are ok with it @rphillips can we close this in favour of 2398?

**Comment by rphillips** [comment]
@yuqi-zhang thanks! closing.

**Pull request closed without merging**